### ISSUE TITLE
--genotype undefined INFO 

### ISSUE THREAD

**Issue by joelmartin** [fritzsedlazeck/Sniffles#26]
using a current build adding --genotype flag produces AF entries in INFO but they aren't defined in the header.  I assume this is allele frequency so am adding
##INFO=<ID=AF,Number=.,Type=Integer,Description="Allele Frequency.">

is that right?

**Comment by fritzsedlazeck** [comment]
Yes its the estimated allele frequency. I will ad it in tomorrow.
Thanks
Fritz
